src/vse_sync_pp/analyzers/ts2phc.py

```diff
@@ -2,24 +2,14 @@
 
 """Analyze ts2phc log messages"""
 
-from .analyzer import Analyzer
+from .analyzer import TimeErrorAnalyzer
 
-class TimeErrorAnalyzer(Analyzer):
+class Ts2PhcAnalyzer(TimeErrorAnalyzer):
     """Analyze time error"""
     id_ = 'ts2phc/time-error'
     parser = id_
     def __init__(self, config):
```

Comment:
The `__init__()` method from `Ts2PhcAnalyzer` is not doing anything beyond the base class.

Reply:
Fixed.